string.c

```diff
@@ -3026,7 +3026,10 @@ rb_str_prepend_multi(int argc, VALUE *argv, VALUE str)
 {
     str_modifiable(str);
 
-    if (argc > 0) {
+    if (argc == 1) {
+	rb_str_update(str, 0L, 0L, StringValue(argv[0]));
```

Comment:
Thanks. I removed it at https://github.com/ruby/ruby/pull/1670/commits/aed74c6e98600cb05d12704d57e57716bfba6c9c